### DISCUSSION TITLE
Multiple sorting in the `VaDataTable` component

### DISCUSSION THREAD

**Discussion by demershov** [epicmaxco/vuestic-ui#3770] [Ideas]
Hi guys,

At the moment, it is possible to sort only by one column, but there are cases when we need to sort by several columns at the same time. 
Will such an opportunity be implemented in the future? 

It seems to me that two separate `props` (`sortBy` and `sortingOrder`) could be replaced with one `prop`, which would accept an object in which the key would be `sortBy`, and the key value would be `sortingOrder`. 
Something like:
`:sort="{ id: 'asc', name: 'desc' }"`

What do you think about it?
Thank you.

**Comment by m0ksem** [comment]
Make sense. We could deprecate `sortBy` and `sortingOrder` in favor of `sort`.

We're not going to update DataTable in the near future. If you're interested in having this feature ASAP I could assist you in making it. Or you can wait like a few months.

**Reply by demershov** [reply]
Thanks for the quick response. 

it takes time to understand the source code, so I will consider your proposal in free time.
